.vale.ini

```diff
@@ -0,0 +1,14 @@
+StylesPath = .github/styles
+
+Vocab = Docs
+
+[*.asciidoc]
+BasedOnStyles = Google, Microsoft, write-good, Vale
+
+Google.Headings = NO
+Google.WordList = NO
+Microsoft.GeneralURL = NO
+Microsoft.Contractions = NO
+Microsoft.HeadingAcronyms = NO
+write-good.TooWordy = NO
```

Comment:
Can you point to the documentation for these? I’m interested in looking what these affect.

Reply:
There's things like Google.WordList suggests "app" everywhere instead of application. But OK, I'll add a comment section to each explaining the reason.

Reply:
Thanks!